fpdcc/ccfp-asset-dashboard
# Add success and error messages to the ajax responses

## Overview

Closes:
- #114 

### Notes
We might want to improve the success messages when the page reloads. The message shows up and the page reloads, which kinda seems like the action is done, but the page is still reloading and we don't want users to use the page during that. Maybe adding the success message near the "save asset" button, plus giving a loading indicator would be more helpful...thoughts?

## Testing Instructions
- review app: https://ccfp-asset-d-ajax-messa-ukywag.herokuapp.com
- test credentials are called "ccfp-asset-dashboard test user"
- search. turn off your wifi network and see an error message.
- select/save an asset. see success message. select/save again with network turned off, see error message.
- delete an asset. see messages etc
